src/libsystemd-network/sd-dhcp6-client.c

```diff
@@ -5,6 +5,7 @@
 
 #include <errno.h>
 #include <sys/ioctl.h>
+#include <net/if.h>
```

Comment:
Maybe a comment next to these includes would be useful? It is not immediately obvious otherwise that there needs to be a specific order